src/PluginRegistrationDialog.cpp

```diff
@@ -0,0 +1,3236 @@
+/**********************************************************************
```

Comment:
This file and one other file duplicate PluginManager.cpp.  There is nothing else in the commit.  You can't tell from this diff viewer, but you can tell by checking out the commit and diff-ing each of the new files against the old one, that they are identical.

The new files are not yet part of the build.